### ISSUE TITLE
Tile Covered by Neighbour Tile by One Line

### ISSUE THREAD

**Issue by peterpwang** [openseadragon/openseadragon#2149]
Hi, I have a site running on OSD viewer. When I zoom in, sometimes a tile is overlapped by one line by its neighbour tile. Here is an example: 

![image](https://user-images.githubusercontent.com/26729808/164339489-1b042904-ee7a-4a77-9812-f8bc668ad8fd.png)

The top strike of the first "8" is covered by the above tile.

I looked into this problem and displayed the position and size of the tiles. Obviously, the bottom of the above tile is over the below title by one line. 
Tile1: y:-102.73317873859304 h:302.7454153239068 bottom: 200.01223658531376
Tile2: y:199.01223658531373

I found if I comment out the line 22426 below, the problem will disappear.

22425     if ( !overlap ) {
22426         //sizeC = sizeC.plus( new $.Point( 1, 1 ) );
22427     }

Please contact me if you need more information.

**Comment by iangilman** [comment]
This is actually kind of a thorny issue. You can see discussion of it here https://github.com/openseadragon/openseadragon/issues/1722 and here https://github.com/openseadragon/openseadragon/pull/2055. Let me know if you're interested in getting involved.

Do you have control over your tile creation? If so, the easiest fix for you is to add a tile overlap of 1, which bypasses the issue.

**Comment by themcoo** [comment]
I think it would be reasonable to give a parameter to opt-out from this behaviour. I too had to patch this forced overlap, as the graphics we are using osd really suffered with annoying shifts appearing:
![image](https://user-images.githubusercontent.com/2964157/214535704-7c61ebe0-192a-47db-9382-06ead2d1c55f.png)

As we only have to support chromium based browsers it would be nice to have a solution without 'npx patch-package' approach.

**Comment by iangilman** [comment]
Indeed... I suppose we could add https://github.com/openseadragon/openseadragon/pull/2055 with an option to turn it on. @themcoo Would you be interested in making a PR for that?
